pstress/generate_mysqld_options.sh

```diff
@@ -0,0 +1,538 @@
+    echoit "  > Adding possible values transactions, statements for option '${OPTION}' to the final list..."
+    echo "${OPTION}=transactions" >> ${OUTPUT_FILE}
+    echo "${OPTION}=statements" >> ${OUTPUT_FILE}
+    echo "${OPTION}=none" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--transaction-isolation" ]; then
+    echoit "  > Adding possible values READ-UNCOMMITTED, READ-COMMITTED for option '${OPTION}' to the final list..."
+    echo "${OPTION}=READ-UNCOMMITTED" >> ${OUTPUT_FILE}
+    echo "${OPTION}=READ-COMMITTED" >> ${OUTPUT_FILE}
+    echo "${OPTION}=REPEATABLE-READ" >> ${OUTPUT_FILE}
+    echo "${OPTION}=SERIALIZABLE" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-doublewrite-files" ]; then
+    echoit "  > Adding possible values 2 , 16 , 126 , 256 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=2" >> ${OUTPUT_FILE}
+    echo "${OPTION}=16" >> ${OUTPUT_FILE}
+    echo "${OPTION}=126" >> ${OUTPUT_FILE}
+    echo "${OPTION}=256" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-tmpdir" ]; then                                          ## fb-mysql
+    echoit "  > Adding possible values null , tmp for option '${OPTION}' to the final list..."
+    echo "${OPTION}=null" >> ${OUTPUT_FILE}
+    echo "${OPTION}=tmp" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--optimizer-trace" ]; then
+    echoit "  > Adding possible values enabled and one_line for option '${OPTION}' to the final list..."
+    echo "${OPTION}=enabled" >> ${OUTPUT_FILE}
+    echo "${OPTION}=one_line" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--performance-schema-instrument" ]; then
+    echoit "  > Adding possible values wait/synch/cond/% for option '${OPTION}' to the final list..."
+    echo "${OPTION}='wait/synch/cond/%=COUNTED'" >> ${OUTPUT_FILE}
+    echo "${OPTION}='wait/synch/cond/%=0'" >> ${OUTPUT_FILE}
+    echo "${OPTION}='wait/synch/cond/%=1'" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--block-encryption-mode" ]; then
+    echoit "  > Adding possible values aes-128-ecb, aes-128-cbc, aes-128-cfb1, aes-192-ecb, aes-192-cbc, aes-192-ofb, aes-256-ecb, aes-256-cbc, aes-256-cfb128 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=aes-128-ecb" >> ${OUTPUT_FILE}
+    echo "${OPTION}=aes-128-cbc" >> ${OUTPUT_FILE}
+    echo "${OPTION}=aes-128-cfb1" >> ${OUTPUT_FILE}
+    echo "${OPTION}=aes-192-ecb" >> ${OUTPUT_FILE}
+    echo "${OPTION}=aes-192-cbc" >> ${OUTPUT_FILE}
+    echo "${OPTION}=aes-192-ofb" >> ${OUTPUT_FILE}
+    echo "${OPTION}=aes-256-ecb" >> ${OUTPUT_FILE}
+    echo "${OPTION}=aes-256-cbc" >> ${OUTPUT_FILE}
+    echo "${OPTION}=aes-256-cfb128" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--default-authentication-plugin" ]; then
+    echoit "  > Adding possible values mysql_native_password, sha256_password for option '${OPTION}' to the final list..."
+    echo "${OPTION}=mysql_native_password" >> ${OUTPUT_FILE}
+    echo "${OPTION}=sha256_password" >> ${OUTPUT_FILE}
```

Comment:
Add one more value to default_authentication_plugin i.e caching_sha2_password
refer this link: https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_default_authentication_plugin